### ISSUE TITLE
Onboarding of SPARC-Relevant Services and Models

### ISSUE THREAD

**Issue by elisabettai** [ITISFoundation/osparc-issues#1093]
Description:
Proactive onboarding of services and models, with focus on SPARC and ANS neuroscience needs (e.g., by contacting authors of SPARC-relevant modeling publications).

Deliverable: 
Services and models on o²S²PARC. The models shall also be published on the Portal. Before onboarding these models, NIH will be contacted to confirm SPARC-relevance

Acceptance criteria:
Targeted number per quarter: 1–2 services, and 2–3 SPARC-relevant models (assuming models can be onboarded using existing services)

Out of scope: –

Deadline:
Continuous

[wrike Y7-MS 3.1.2: Onboarding of SPARC-Relevant Services and Models](https://www.wrike.com/open.htm?id=1155713395)

```[tasklist]
### Eisbock
- [ ] https://github.com/ITISFoundation/osparc-issues/issues/1608
- [ ] https://github.com/ITISFoundation/osparc-issues/issues/1609
```

```[tasklist]
### Tasks
- [ ] Jupyterlab with Julia
- [ ] Team 6 Codeathon (Dashboard+ChatGPT)
- [ ] New version sim_gui-app/web (deprecated)
- [ ] New version tvb-app/web (deprecated)
- [ ] Port jupyter-fenics to dy-sidecar
- [ ] #998
```

```[tasklist]
### Quilmes
- [x] EIT/Kirill's use-case: Update EIDORS Service @elisabettai
- [x] EIT/Kirill's use-case: Additional mesher service? @elisabettai
- [x] Sergey LU: Make 3 services out of Sergey Makaroff's Matlab scripts
```

```[tasklist]
### Kobayashi Maru
- [ ] https://github.com/ITISFoundation/osparc-issues/issues/1188
- [ ] https://github.com/ITISFoundation/osparc-issues/issues/1211
- [x] Viewer for OA publication
```

**Comment by elisabettai** [comment]
For regulatory-grade modelling, we can create a service for [AxonDeepSeg](https://axondeepseg.readthedocs.io/en/latest/documentation.html#graphical-user-interface-gui).

**Comment by elisabettai** [comment]
### Update [Kobayashi Maru](Kobayashi_Maru)
- New service: AxonDeepSeg GUI (see #1188 )
- Supplement for publication, including 2 new services (1 created by the publications author): [Athavale et al., 2024](https://iopscience.iop.org/article/10.1088/1741-2552/ad1610): link to oSPARC template: https://osparc.io/study/1170826c-9a64-11ee-8b3c-02420a0b1983
- New version of jupyter-pysonic: ported to dy-sidecar and added MorphoSONIC library (see #1211)

**Comment by elisabettai** [comment]
## Update [This is Sparta](https://github.com/ITISFoundation/osparc-issues/blob/master/reviews/20240202_This_Is_Sparta.md)
### In progress
- Jupyterlab with AxonDeepSeg (related to https://github.com/ITISFoundation/osparc-issues/issues/1188 ), also used for testing a cookiecutter developed for

**Comment by elisabettai** [comment]
## Update Sprint Schoggilebe
### Done
- Jupyterlab with AxonDeepSeg with GPU support (related to #1188)
- AxonDeepSeg Runner (computational service for image segmentation using AxonDeepSeg (related to #1188)

**Comment by elisabettai** [comment]
Y7Q3 Status update (05/24/2024): Continuously Ongoing
Implementation: 
Three new JupyterLab-based services have been created and published to all o²S²PARC users. They are also accessible via the dedicated o²S²PARC [page on the SPARC Portal](https://sparc.science/tools-and-resources/4LkLiH5s4FV0LVJd3htsvH). 

JupyterLab SPARC: an interactive scripting environment for Python coding, with pre-installed, SPARC-relevant libraries - in particular the SPARC Python Client and other Python packages of interest that were identified in the past by Charles Horn (see [Docker_SPARC](https://github.com/cchorn/Docker_SPARC) project). This new Service was also utilized as part of a live demo at the [SPARC Python Client webinar](https://sparc.science/news-and-events/events/sparc-python-client-webinar) and it will be promoted at the upcoming SPARC FAIR Codeathon in August.

JupyterLab Julia: an interactive scripting environment for coding with Julia, a programming language that is becoming popular among scientists. This was for example requested to enable faster execution of the[ Cardiovascular System model](https://doi.org/10.26275/5yvu-tr0d) previously published on the SPARC Portal. 

JupyterLab R: an interactive scripting environment for coding in R, a programming language used for statistical analysis, and prominent in genomics and transcriptomics analysis (see for example this [SPARC transcriptomics dataset](https://doi.org/10.26275/hrww-enzr)).

In addition, two new services have been released that are part of the growing o²S²PARC meta-modelling framework:

o²S²PARC Meta Parallel Runner: allows users to scalably run Studies in parallel on the platform (e.g. for iterating over a parameter set). Users only need to specify the ID of the Study, the number of parallel jobs to start and an input file with the parameters sets to evaluate.

o²S²PARC Meta Dakota: allows to leverage Dakota - a software for optimization, surrogate modeling, uncertainty propagation, and more - in combination with the o²S²PARC Meta Parallel Runner.

**Comment by mrnicegyu11** [comment]
Closing this in favor of current Y8 task https://github.com/ITISFoundation/osparc-issues/issues/1646
